### DISCUSSION TITLE
Question about queue mode

### DISCUSSION THREAD

**Discussion by jreyesr** [jreyesr/n8n-nodes-carbonejs#8] [Q&A] [answered]
Asked by @Vladimir-Va on issue #7, then moved here:

> Hello, I'm experimenting with the carbone js node and getting an error in the topic.
> Is this possible if n8n works in queue mode?
> 
> ```
> To avoid this, change the N8N_DEFAULT_BINARY_DATA_MODE environment variable to filesystem. This causes n8n to save data to > disk, instead of using memory.
> 
> If you're using queue mode, keep this to default. n8n doesn't support filesystem mode with queue mode.
> ```

**Comment by jreyesr** [comment]
@Vladimir-Va: Is there any more to the error message that you saw? I think I remember seeing it on my instances, but it was something about N8N running out of memory when handling large files or something. Could you perhaps provide a screenshot of the error or whatever is nearby the two lines that you provided, please?

**Comment by Vladimir-Va** [comment, marked as answer]
Thank you for support.
After update node to latest 1.1.2 rendering is started!
